src/data/staking-products.json

```diff
@@ -636,6 +636,35 @@
         "eventAction": "Clicked",
         "eventName": "Clicked Sensei Node go to link"
       }
+    },
+    {
+      "name": "Figment",
+      "imageName": "Figment",
+      "hue": 52,
+      "launchDate": "2021-12-31",
```

Comment:
@ivanszeftel This won't end up affecting anything, since anything over 1 year is weighted the same, but you mentioned "mid 2021" which didn't let me zero it in on the exact date, so I just defaulted to the end of that year. Please feel free to update to proper date or more accurate month if you'd like; supporting evidence always appreciated =)